contents/blog/how-to-fix-your-growth-team.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Why your growth team isn't winning (and how to fix it)
+date: 2023-09-14
+author: ["lior-neu-ner"]
+showTitle: true
+rootpage: /blog
+sidebar: Blog
+hideAnchor: true
+featuredImage: ../images/blog/athlete-hog.jpeg
+featuredImageType: full
+category: Product growth
+tags: 
+  - Guides
+  - Product metrics
+  - Product engineers
+---
+
+There are tell-tale signs when a growth team isn't succeeding: Stuck metrics, missed KPIs, and burnt-out engineers. Everyone is frustrated, but how do you fix it?
+
+[Alexey Komissarouk](https://alexeymk.com/), former head of growth engineering at [MasterClass](https://www.masterclass.com/), has seen it all. He grew Masterclass xyz... 
+
+In this post, Alexey shares his advice on how to get your growth team winning again.
+
+TLDR; there are 3 areas to investigate:
+
+1. Your team's velocity.
```

Comment:
one piece that's missing here is why these three categories are relevant.

Experiment Velocity (Engineer-weeks per experiment) * Experiment Win Rate * Average Experiment Win Size = your team's impact per engineer

IE, 2 eng-weeks per experiment * 40% win rate * 5% avg win size = 1% impact per eng-week.

So if you have 3 engineers and a quarter (12 weeks) you expect to move the metric by 36% that quarter 

But if you boost your win rate to 50%, it goes to 45%, or if get to 1.5 eng-weeks per experiment, the impact goes to  48%.

(Keep in mind that in practice, you'll want to roughly halve that estimate due to Winner's Curse https://medium.com/airbnb-engineering/selection-bias-in-online-experimentation-c3d67795cceb)

Reply:
Thanks! I want to jump straight into the post, so I think I'll just remove this section